examples/advanced/tidb-cluster.yaml

```diff
@@ -82,50 +282,191 @@ spec:
+    # imagePullPolicy: IfNotPresent
+    # ImagePullSecrets: secretName
+    # hostNetwork: false
+    # serviceAccount: advanced-tidb-tidb
+    # priorityClassName: system-cluster-critical
+    # schedulerName: tidb-scheduler
+    # nodeSelector:
+    #   app.kubernetes.io/component: tidb
+    # annotations:
+    #   node.kubernetes.io/instance-type: some-vm-type
+    # tolerations:
+    #   - effect: NoSchedule
+    #     key: dedicated
+    #     operator: Equal
+    #     value: tidb
+    # configUpdateStrategy: RollingUpdate
+    # statefulSetUpdateStrategy: RollingUpdate
+
+    ## List of environment variables to set in the container
+    ## Note that the following env names cannot be used and may be overrided by TiDB Operator builtin envs
+    ##   - NAMESPACE
+    ##   - TZ
+    ##   - SERVICE_NAME
+    ##   - PEER_SERVICE_NAME
+    ##   - HEADLESS_SERVICE_NAME
+    ##   - SET_NAME
+    ##   - HOSTNAME
+    ##   - CLUSTER_NAME
+    ##   - POD_NAME
+    ##   - BINLOG_ENABLED
+    ##   - SLOW_LOG_FILE
+    ## Ref: https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/
+    # env:
+    #   - name: MY_ENV_1
+    #     value: value1
+    #   - name: MY_ENV_2
+    #     valueFrom:
+    #       fieldRef:
+    #         fieldPath: status.myEnv2
+
+    ## Custom sidecar containers can be injected into the TiDB pods,
+    ## which can act as a logging/tracing agent or for any other use case
+    # additionalContainers:
+    #   - name: myCustomContainer
+    #     image: ubuntu
+
+    ## custom additional volumes in TiDB pods
+    # additionalVolumes:
+    #   - name: nfs
+    #     nfs:
+    #       server: 192.168.0.2
+    #       path: /nfs
+
+    ## custom additional volume mounts in TiDB pods
+    # additionalVolumeMounts:
+    #   - name: nfs
+    #     mountPath: /nfs
+
+    ## define the time period between pod Terminating and killed
+    ## Ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-execution
+    # terminationGracePeriodSeconds: 30s
+
+    ## defines privilege and access control settings for TiDB pods
+    ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+    # podSecurityContext:
+    #   sysctls:
+    #     - name: net.ipv4.tcp_syncookies
+    #       value: 0
+
+    ## prob tidb-server readiness
+    ## valid type values:
+    ##   - `tcp`, which uses Kubernetes TCPSocketAction to prob the 4000 tcp port of the pod
+    ##   - `command`, which uses curl to access the /status path on port 10080 of the pod
+    ## This is only supported after tidb v4.0.9, ref: https://github.com/pingcap/tidb/pull/20694
+    # readinessProbe:
+    #   type: command
+
+    ## when enabled, TiDB will accept TLS encrypted connections from MySQL client
+    ## Ref: https://docs.pingcap.com/tidb-in-kubernetes/stable/enable-tls-for-mysql-client
+    # tlsClient:
+    #   enabled: true
+
+    ## MANUAL CONFIG NOT RECOMMENDED
```

Comment:
This is only for `binlogEnabled `? If yes, please move this to the next line, it seems that all the parameters following this line are not recommended to change manually.

Reply:
yes, it's only for `binlogEnabled`.
moved.